var/spack/repos/builtin/packages/py-dgl/package.py

```diff
@@ -101,8 +122,11 @@ def install(self, spec, prefix):
             args = std_pip_args + ["--prefix=" + prefix, "."]
             pip(*args)
 
-        # Work around installation bug: https://github.com/dmlc/dgl/issues/1379
-        install_tree(prefix.dgl, prefix.lib)
+        # Workaround causes errors in @1.0.1 during install phase:
+        # Error: OSError: No such file or directory: '<prefix>/dgl'
```

Comment:
Another way to phrase this would be that old versions did not install correctly, but new versions do.